INSTALL.md

```diff
@@ -17,39 +17,40 @@ Install tools and dependencies used for development:
     $ yum install git ant ant-devel java-1.6.0-openjdk java-1.6.0-openjdk-devel
     mysql mysql-server mkisofs gcc python MySQL-python openssh-clients wget
 
-    # yum -y update
-    # yum -y install java-1.7.0-openjdk
-    # yum -y install java-1.7.0-openjdk-devel
-    # yum -y install mysql-server
-    # yum -y install git
-    # yum -y install genisoimage
+    $ yum -y update
```

Comment:
this is not valid. the prompt you could run yum from on most syustems would be '#'